python/ray/dag/compiled_dag_node.py

```diff
@@ -592,6 +593,8 @@ def __init__(
                 are allowed to be sent to this DAG. Before submitting more requests,
                 the caller is responsible for calling ray.get to get the result,
                 otherwise, RayAdagCapacityExceeded is raised.
+            custom_nccl_group: The default custom NCCL group to be used for P2P NCCL
```

Comment:
What is the meaning of "the default custom"? Can you elaborate on this?

Reply:
"Default" means it is used as the default NCCL group for P2P NCCL communications whose `transport="nccl"`. "Custom" means it is provided by the user.
Changed comment to: custom NCCL group to be used for P2P NCCL communications whose `transport="nccl"`.

Reply:
Changed name to `default_nccl_group`. Changed comment to: default NCCL group to be used...